{
    "affected": [
        {
            "package": {
                "ecosystem": "photon:3.0",
                "name": "linux-aws",
                "purl": "pkg:rpm/vmware/linux-aws?distro=photon-3"
            }
        },
        {
            "package": {
                "ecosystem": "photon:3.0",
                "name": "linux-rt",
                "purl": "pkg:rpm/vmware/linux-rt?distro=photon-3"
            }
        },
        {
            "package": {
                "ecosystem": "photon:3.0",
                "name": "linux-esx",
                "purl": "pkg:rpm/vmware/linux-esx?distro=photon-3"
            }
        },
        {
            "package": {
                "ecosystem": "photon:3.0",
                "name": "linux-secure",
                "purl": "pkg:rpm/vmware/linux-secure?distro=photon-3"
            }
        },
        {
            "package": {
                "ecosystem": "photon:3.0",
                "name": "runc",
                "purl": "pkg:rpm/vmware/runc?distro=photon-3"
            }
        },
        {
            "package": {
                "ecosystem": "photon:3.0",
                "name": "linux",
                "purl": "pkg:rpm/vmware/linux?distro=photon-3"
            }
        }
    ],
    "id": "PHSA-2021-3.0-0345",
    "modified": "2023-03-16T05:25:30Z",
    "published": "2019-03-06T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-3.0-3"
        }
    ],
    "related": [
        "CVE-2020-36385",
        "CVE-2021-43784"
    ]
}